capstone-frontend/src/components/BookListAddDropdown.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { DropdownButton, Dropdown } from 'react-bootstrap';
+import '../styles/Book.css';
+
+const addBookToBookList = async (bookId, bookListJson) => {
+
+  const bookListUpdateJson = {
+    "id": bookListJson.id,
+    "add_gbookIDs": bookId,
+  }
+
+  // Update BookList in Firebase
+  fetch("/api/booklist", {
+    method: "PUT",
+    body: JSON.stringify(bookListUpdateJson)
+  });
+}
+
+const BookListAddDropdown = ({ book, userBookLists }) => {
+  if (userBookLists.length > 0) {
+    return (
+      <DropdownButton id="dropdown-list-add" className="btn-margin center-horizontal" title="Add to List">
+        {
+          userBookLists.map(bookList =>
+            <Dropdown.Item key={bookList.id}
+              onSelect={() => addBookToBookList(book.id, bookList.id)}>
+              {bookList.name}
+            </Dropdown.Item>
+          )
+        }
+      </DropdownButton>
+    );
+  } else {
+    return (
+      <DropdownButton id="dropdown-list-add" className="btn-margin center-horizontal" title="No Lists Found" variant="warning">
```

Comment:
Maybe change this to "Create a List" to make it more obvious that the user can click this button to get started with their first list. Especially not that we don't have an easily accessibly "Create a List" page. I'm also thinking we should add a Create a List button in the MyLists page, similar to the one in the My Clubs page.

Reply:
I agree with this! Now it is just a "Create New List" button instead of dropdown